10-23 15:56:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-172]
10-23 15:58:10 pageStore: Transaction log could not be truncated; size: 16 MB
10-23 21:23:01 jdbc[156]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:582)
	at java.sql.DriverManager.getConnection(DriverManager.java:185)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:339)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:405)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
10-24 14:07:29 jdbc[470]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:582)
	at java.sql.DriverManager.getConnection(DriverManager.java:185)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:339)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:405)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
